### PR TITLE
Postgres attempt

### PR DESCRIPTION
**Pull request by gigicupp**
@christulin @amosNash 
Hey team,

This one is a really small pull request. I made a small change to the README and created a new function that gets me existing images from my S3 and add a random image to a user during seeding.

thank you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ GET
 Endpoint: http://localhost:5007/users/:userId
 
 Returns a JSON object with user info. For example:
-{"languages":["doggo","duck","chicken"],"_id":"60443ea4ad8703830eeff43d","userId":10201,"__v":0,"avatarUrl":"https://images.app.goo.gl/dJzfRGB8Fz4DS7or9","bio":"I am the cutest little doggo ever.","identityVerified":false,"isSuperhost":false,"joinDate":"2021-03-07T02:49:28.287Z","name":"LucyFur","responseRate":10,"responseTime":"within a month"}
+```{"languages":["doggo","duck","chicken"],"_id":"60443ea4ad8703830eeff43d","userId":10201,"__v":0,"avatarUrl":"https://images.app.goo.gl/dJzfRGB8Fz4DS7or9","bio":"I am the cutest little doggo ever.","identityVerified":false,"isSuperhost":false,"joinDate":"2021-03-07T02:49:28.287Z","name":"LucyFur","responseRate":10,"responseTime":"within a month"}```
```
Comment (amosNash) [inline README.md:48]: I just have a question about what the three back ticks are doing? What is the purpose of them?

File: server/database/seed.js [hunk 2]
```diff
@@ -1,21 +1,39 @@
 require('dotenv').config();
 const { generatePhoto } = require('./helpers');
 const faker = require('faker');
-const AWS = require('./aws.js');
+const { AWS, S3 } = require('./aws.js');
 const { mongoose, User } = require('./schema.js');
 
 const languages = ['English', 'Spanish', 'French', 'Portuguese', 'German', 'Italian', 'Cambodian', 'Thai', 'Shyriiwook'];
```
Comment (amosNash) [inline server/database/seed.js:7]: Is this line necessary?

File: server/database/seed.js [hunk 3]
```diff
@@ -1,21 +1,39 @@
...
 
+let params = { Bucket: 'userservicebucket' };
+
+let listAllKeys = S3.listObjectsV2(params).promise();
+
+const getImgKeys = () => {
```
Comment (amosNash) [inline server/database/seed.js:13]: This looks good!

File: server/database/seed.js [hunk 4]
```diff
@@ -56,5 +74,5 @@ const seedManyUsers = (start, number) => {
     .catch(() => console.error('Batch unsuccessful'));
 };
 
-seedManyUsers(200, 10000);
+seedManyUsers(10206, 10);
```
Comment (amosNash) [inline server/database/seed.js:76]: Why are you choosing to input 10206 and 10?

**Review by amosNash — COMMENTED** [review]
Looks good! I just had a few questions.

File: README.md (hunk 1 above)
Comment (gigicupp) [reply README.md:48]: Apparently the 3 back ticks make easier to read.

File: server/database/seed.js (hunk 2 above)
Comment (gigicupp) [reply server/database/seed.js:7]: yes

File: server/database/seed.js (hunk 4 above)
Comment (gigicupp) [reply server/database/seed.js:76]: oh, i was just playing with existing data and adding on to it 10 at a time after seeding with 10,000.

**Pull request merged**